wiki/introduction/setting-up.md

```diff
@@ -21,10 +21,12 @@ repository or use GitHub's template feature to generate a mod that you can use
 straight away.
 
 To download the ZIP file for the template mod, open the [GitHub repository](https://github.com/QuiltMC/quilt-template-mod),
-then go to the `< >` button and press `Download ZIP`, or refer to the following images:
+then go to the `< >` button and press `Download ZIP`, or refer to the following image:
 
-![Arrow pointing to the <> button](/introduction/setting-up-1.png)
-![Arrow pointing to the Download ZIP button](/introduction/setting-up-2.png)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="/introduction/setting-up-1-dark.png">
+  <img alt="The GitHub repository with an arrow pointing to the code button, and another to the Download ZIP button inside the opened popup" src="/introduction/setting-up-1-light.png">
+</picture>
```

Comment:
I feel like we should do that automatically in HTML processing (Rehype) at some point, but this will work for know.